README.rst

```diff
@@ -55,6 +55,8 @@ Next, let's parse something and get an abstract syntax tree:
         <Node called "bold_open" matching "((">
         <RegexNode called "text" matching "bold stuff">
         <Node called "bold_close" matching "))">
+    >>> print grammar['text'].parse('other stuff')
```

Comment:
I think this should be removed. It clutters up the simple use case example. Once the documentation is expanded then we can better document use cases like this.

Reply:
Please remove this.